alpha-build-machinery/make/examples/multiple-binaries/manifests/operator.openshift.io_myoperatorresources.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: myoperatorresources.operator.openshift.io
+spec:
+  group: operator.openshift.io
+  names:
+    kind: MyOperatorResource
+    plural: myoperatorresources
+  scope: ""
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  "validation":
```

Comment:
@sttts any idea why the quotes around `"validation"` and `"openAPIV3Schema"`?

Reply:
Are they also in my PR?

Reply:
@sttts They're not, but testing this it looks like it only puts quotes if these sections weren't already there. Possibly a controller-tools bug?